pkg/runtime/wfengine/wfengine.go

```diff
@@ -53,6 +54,7 @@ const (
 
 var (
 	wfLogger            = logger.NewLogger("dapr.runtime.wfengine")
+	wfBackendLogger     = logger.NewLogger("wfengine.backend")
```

Comment:
Good idea

Reply:
(For other reviewers) This helped figuring out where the logs were coming from, since the dtf-go library uses this logger too